Composer/packages/client/__tests__/pages/botProjectsSettings/AppIdAndPassword.test.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
@liweitian

The following are some things Im seeing in this branch.

1.  Create a root bot and add two skills to it. Update the LUIS subscription key or Qna subscription key on the root bot.

*   Expected: When i start any of the skills, the luis or qna config  needs to be populated with the LUIS key from the root bot
*   Actual: The LUIS/ Qna key is updated only on the root bot. Its not progressed to the skill

2\. With the latest SDK runtime, Microsoft AppId and Microsoft App Password are not required to call skills anymore. We can safely remove the styles. The error ribbon needs to appear for missing LUIS key or QNA key if the bot needs those two configs. Currently those errors dont show up.